### PR TITLE
add a benchmark testcase for measuring the time of creating pod and a container

### PR DESCRIPTION
**Pull request by lifupan**
add a benchmark testcase for measuring the time of creating pod and a container, plus some fixes.

### CODE REVIEW

**Comment by lifupan** [comment]
I signed it

**Comment by lifupan** [comment]
Hi @Random-Liu
It seems there is one testcase failed in travis, but I cannot find what's the root cause.
Could you have a look?

File: pkg/benchmark/README.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+sudo sh -c "echo 'deb http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/master/xUbuntu_$(lsb_release -rs)/ /' > /etc/apt/sources.list.d/kata-containers.list"
+curl -sL  http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/master/xUbuntu_$(lsb_release -rs)/Release.key | sudo apt-key add -
+sudo -E apt-get update
+sudo -E apt-get -y install kata-runtime kata-proxy kata-shim
+```
+### Install kata shimv2
+```sh
+go get github.com/kata-containers/runtime
+pushd $GOPATH/src/github.com/kata-containers/runtime
+git remote add hyper https://github.com/hyperhq/kata-runtime
```
[the remark this answers is not in the document]
Comment (lifupan) [reply pkg/benchmark/README.md]: Yeah, nice catch. I forgot to update it to master.
Updated.

File: pkg/benchmark/README.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+make 
+sudo -E PATH=$PATH make install
+popd
+
+sed -i 's/image =/#image =/' /usr/share/defaults/kata-containers/configuration.toml
+```	
+### Install critest
+
+```sh
+go get https://github.com/lifupan/cri-tools.git
```
[the remark this answers is not in the document]
Comment (lifupan) [reply pkg/benchmark/README.md]: Updated.

File: pkg/benchmark/README.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
+# pod and container lifecycle benchmark for kata
```
Comment (feiskyer) [inline pkg/benchmark/README.md]: could we move this into kata repo instead?

**Comment by gnawux** [comment]
This PR is not Kata-specific, on the contrary, it introduces some more cases, which are cared about more in the multi-runtime scenarios.

For the README file, it described how to test with Kata Containers, which is not directly related with the cases, and should be claimed as an example.

File: pkg/benchmark/README.md (hunk 3 above)
Comment (lifupan) [reply pkg/benchmark/README.md]: Just as @gnawux said, it is not suitable to place a kata specific README here, thus I remove it.

Comment (feiskyer) [reply pkg/benchmark/README.md]: Thanks, other parts lgtm

**Comment by lifupan** [comment]
Hi @Random-Liu  Any comments?

**Pull request merged**